cosmwasm/contracts/yield-ward/src/execute/reinit.rs

```diff
@@ -86,8 +90,9 @@ pub fn handle_reinit(
         bank_transfer_msg,
         Event::new("unstake_finished")
             .add_attribute("unstake_id", reinit_unstake_id.to_string())
-            .add_attribute("token", deposit_token_denom)
-            .add_attribute("lp_amount", unstake_item.lp_token_amount)
+            .add_attribute("chain", token_config.chain.clone())
+            .add_attribute("yield_contract", token_config.evm_yield_contract.clone())
```

Comment:
I'm not sure, how the `Into<String>` works exactly for `&String`, but I guess it'll never be worse than plain cloning, so I suggest replacing all the `.add_attribute("attr_name", attr_string.clone())` with `.add_attribute("attr_name", &attr_string)` for the code to be cleaner and possibly slightly optimizing the performance